src/Stripe.net/Services/_base/Service.cs

```diff
@@ -245,6 +261,91 @@ protected TEntityReturned CreateEntity(BaseOptions options, RequestOptions reque
                 cancellationToken).ConfigureAwait(false);
         }
 
+#if !NET45
+        protected IEnumerable<T> ListRequestAutoPaging<T>(
+            string url,
+            ListOptions options,
+            RequestOptions requestOptions)
+            where T : IStripeEntity
+        {
+            return this.ListRequestAutoPagingAsync<T>(url, options, requestOptions)
+                .ToEnumerable();
+        }
+
+        protected async IAsyncEnumerable<T> ListRequestAutoPagingAsync<T>(
+            string url,
+            ListOptions options,
+            RequestOptions requestOptions,
+            [EnumeratorCancellation] CancellationToken cancellationToken = default)
```

Comment:
This is not very well documented, but what the [`EnumeratorCancellation`](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.enumeratorcancellationattribute?view=netcore-3.1) attribute does is add support for two different ways of passing the cancellation token:
- directly as a normal method parameter
- using the [`.WithCancellation()`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.threading.threadingtools.withcancellation?view=visualstudiosdk-2019) extension method